### PR TITLE
view info

### LINKED ISSUE #9: info

### PR DESCRIPTION
**Pull request by ibednorz**
simple info view, mobile view to improve, but we need to determine what about the popup - we increase it permanently or add classes that will modify it

### CODE REVIEW

File: src/app/app.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
+import infoContentProvider from './components/info'
```
Comment (pdybowski) [inline src/app/app.ts]: Just components

File: src/app/app.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 import { playAudio } from './utils'
 import { GamePage, HomePage } from './screens'
 
 export class App {
   start() {
     console.log('Starting application...')
+    new infoContentProvider()
```
Comment (bartoszbialecki) [inline src/app/app.ts]: Why is it initialized here? I think it belongs to start page so it should be used there.

File: src/app/components/info/index.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
+import { renderPopup } from '../popup'
+import { createElement } from '../../utils/createElement'
+
+const btn = document.querySelector('#info-button')
```
Comment (bartoszbialecki) [inline src/app/components/info/index.ts]: Maybe move it to the constructor?

**Review by pdybowski — CHANGES_REQUESTED** [review]

File: src/app/components/info/index.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
+import { renderPopup } from '../popup'
+import { createElement } from '../../utils/createElement'
+
+export default class infoContentProvider {
+  constructor() {
+    const btn = document.querySelector('#info-button')
+    btn?.addEventListener('click', this.render.bind(this))
+  }
+
+  render() {
```
Comment (pdybowski) [inline src/app/components/info/index.ts]: check private

File: src/app/components/info/index.ts [hunk 5]
```diff
@@ -0,0 +1,120 @@
+import { renderPopup } from '../popup'
+import { createElement } from '../../utils/createElement'
+
+export class infoContentProvider {
```
Comment (pdybowski) [inline src/app/components/info/index.ts]: First letter should be uppercase in classes

File: src/app/screens/homepage/index.ts [hunk 6]
```diff
@@ -74,5 +75,6 @@ export class HomePage {
     this.triggerInfoButton()
     this.triggerStartButton()
     this.triggerSettingsButton()
+    new infoContentProvider()
```
Comment (pdybowski) [inline src/app/screens/homepage/index.ts]: First letter - uppercase

File: src/app/screens/homepage/index.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import { GamePage } from '../gamepage'
+import {infoContentProvider} from '../../components'
```
Comment (pdybowski) [inline src/app/screens/homepage/index.ts]: First letter uppercase

**Review by pdybowski — CHANGES_REQUESTED** [review]
I will clean this code later. Make first letter of a class name to uppercase

**Review by pdybowski — APPROVED** [review]

**Review by pdybowski — APPROVED** [review]

**Review by hoolek77 — APPROVED** [review]

**Review by aleksandraj036 — APPROVED** [review]

**Pull request merged**